packages/components/src/navigator/stories/index.tsx

```diff
@@ -46,7 +47,7 @@ const Template: ComponentStory< typeof NavigatorProvider > = ( {
 				<CardBody>
 					<p>This is the home screen.</p>
 
-					<HStack justify="flex-start" wrap>
+					<VStack alignment="left">
```

Comment:
Noticed that `alignment="left"` doesn't work as expected in VStack. It seems `useFlex` forces `alignItems` to be "normal" regardless of the value provided by the `useHStack` hook (for the VStack case). I think that's probably a bug in `useFlex` that I'll explore separately.

Reply:
Thank you for spotting. Let us know if you have any further findings!